rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -45,12 +45,14 @@ install -m 755 -d %{buildroot}/usr/share/%{name}/scripts
 install -m 644 dom0/*.sls %{buildroot}/srv/salt/
 install -m 644 dom0/*.top %{buildroot}/srv/salt/
 install -m 644 dom0/*.j2 %{buildroot}/srv/salt/
+install -m 644 dom0/*.yml %{buildroot}/srv/salt/
 install -m 644 dom0/securedrop-update %{buildroot}/srv/salt/
 install -m 644 dom0/securedrop-login %{buildroot}/srv/salt/
 install -m 644 dom0/securedrop-launcher.desktop %{buildroot}/srv/salt/
 install -m 655 dom0/securedrop-handle-upgrade %{buildroot}/srv/salt/
 # The next file should get installed via RPM not via salt
 install -m 755 dom0/securedrop-update %{buildroot}/srv/salt/securedrop-update
+install -m 755 dom0/securedrop-admin %{buildroot}/%{_bindir}/securedrop-admin
```

Comment:
Instead of:

```
install -m 755 dom0/securedrop-admin %{buildroot}/%{_bindir}/securedrop-admin
```

why not

```
install -m 755 scripts/securedrop-admin.py %{buildroot}/%{_bindir}/securedrop-admin
```

? N.B. Haven't tested, mostly questioning the need for the wrapper script indirection, as above.